docs(gen): tutorials requeriments update part ii (#2924)
scaleway/docs-content: tutorials/foreman-puppet/index.mdx

<Lightbox src="scaleway-foreman_login.webp" alt="" />

## Foreman dashboard

After login, the Foreman dashboard displays an overview of your Puppet environment.

> It displays a summary of "Host Configuration Status", showing the number of Puppet agent nodes and their respective statuses. At the first connection, one single Instance should be visible in the "good host" section.

Reviewer: ```suggestion
A summary of "Host Configuration Status" displays, with the amount of Puppet agent nodes and their statuses. Upon first connection, one single Instance should be visible in the "good host" section.
```
